### PR TITLE
memcache unit test

### PR DESCRIPTION
**Pull request by nigrosimone**
memcache unit test

### CODE REVIEW

File: tests/test-object-cache-memcache.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Class W3_Object_Cache_Memcache_Test
+ *
+ * @package W3_Total_Cache
+ */
+require_once dirname(__FILE__) . '/w3-object-cache-test.php';
```
Comment (rabin-io) [inline tests/test-object-cache-memcache.php]: why no use __DIR__ instead of calling dirname one __FILE__ ?

**Comment by nigrosimone** [comment]
Mhh,  _dir_ is php >= 4 costant... test start from php 3

**Comment by nigrosimone** [comment]
I'm wrong,  from docs

> 5.3.0	Added __DIR__ and __NAMESPACE__ constants

**Comment by rabin-io** [comment]
what do you mean by 
> test start from php 3
do you run the test from PHP version 3 ?

**Pull request merged**